q2_feature_table/tests/filter/test_filter_samples.py

```diff
@@ -363,8 +414,8 @@ def test_where(self):
                       ['O1', 'O2'],
                       ['S1', 'S2', 'S3'])
         where = "Subject='subject-1' AND Subject='subject-2'"
-        actual = filter_samples(table, metadata=metadata, where=where)
-        self.assertTrue(actual.is_empty())
+        with self.assertRaisesRegex(ValueError, 'table is empty'):
```

Comment:
Same as above comments for comment on line 408 - additionally, you'll want to include the True case for this test as well.